### ISSUE TITLE
Error thrown if there is no `dist` directory

### ISSUE THREAD

**Issue by stuartjnelson** [vladshcherbin/rollup-plugin-copy#9]
- NPM: `5.6.0`
- Node: `8.6.0
- OS: MacOS Sierra 10.12.6
- Rollup : `0.52.0`
- Rollup Plugin Copy: `0.2.3`

When I run `npm run build` and there is no `dist` directory I get this error;

```
(rollup-plugin-copy) 'src/css/badger-accordion.css' -> 'dist/badger-accordion.css' (✘)

    Error: ENOENT: no such file or directory, chmod 'path/to/plugin/badger-accordion/dist/badger-accordion.css'
npm ERR! code ELIFECYCLE
npm ERR! errno 254
```

Sometimes I get the error twice... It still copies the files over so it works just it's throwing an error... Thanks for making this plugin it's just what I needed! Let me know if I can help getting this fixed.

**Comment by ProLoser** [comment]
Duplicate of #7

**Comment by SerafimovichEugene** [comment]
doc example doesn't work for me: 
` copy({        
            "node_modules/bootstrap/dist": "dist/vendor/bootstrap",
            verbose: true
        })`
workaround: 
` copy({        
           "./src/styles/fonts/": "./dist/fonts/",
            verbose: true
        })`

**Comment by marverix** [comment]
https://github.com/marverix/rollup-plugin-copier In version 1.1.0 I have added `createPath` flag

**Comment by vladshcherbin** [comment]
This error is not because there is no `dist` directory, it's because the file you want to copy (src/css/badger-accordion.css) doesn't exist, which is written in error message.

Closing as duplicate of #7

**Comment by vladshcherbin** [comment]
I believe this is fixed in 1.0.0. Try it and leave a comment if error is still there.
